Gemfile

```diff
@@ -42,7 +42,9 @@ gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
 gem 'staccato'
 gem 'twilio-ruby'
-gem 'two_factor_authentication', github: 'Houdini/two_factor_authentication'
+# TODO(amos): Unfork this gem: https://github.com/18F/identity-private/issues/708
+gem 'two_factor_authentication', github: 'amoose/two_factor_authentication',
+                                 branch: 'feature/more_options'
```

Comment:
I think this can be done later. I don't think it would be trivial to make this change now. I would vote for merging this once Amos addresses the other feedback and then I'll start working on the refactor that removes the upstream change.